config/initializers/omniauth.rb

```diff
@@ -73,4 +87,45 @@ def self.bypass?
     scope: ["email", "openid", "profile", "dqt:read"],
     send_scope_to_token_endpoint: false
   }
+
+  unless OneLoginSignIn.bypass?
+    provider :openid_connect, {
+      name: :onelogin,
+      callback_path: "/auth/onelogin",
+      client_auth_method: "jwt_bearer",
+      client_options: {
+        host: onelogin_sign_in_issuer_uri&.host,
+        identifier: ENV["ONELOGIN_SIGN_IN_CLIENT_ID"],
+        port: onelogin_sign_in_issuer_uri&.port,
+        redirect_uri: onelogin_sign_in_redirect_uri&.to_s,
+        scheme: onelogin_sign_in_issuer_uri&.scheme,
+        secret: onelogin_sign_in_secret_key
+      },
+      discovery: true,
+      issuer: ENV["ONELOGIN_SIGN_IN_ISSUER"],
+      response_type: :code,
+      scope: %i[openid email phone],
+      send_scope_to_token_endpoint: false
```

Comment:
I think we need to send the `vtr` parameter as `["Cl.Cm"]` according to https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/authenticate-your-user/#make-a-request-for-authentication-and-identity

Reply:
that parameter is optional.
"If you do not specify the vtr parameter, your service will automatically log your users in at the medium level of authentication (Cl.Cm)."
https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/authenticate-your-user/#replace-the-placeholder-values-in-your-example

I'm happy to put that parameter in, to make it explicit.

Reply:
great, thanks for clarifying.